src/content/docs/codestream/codestream-ui-overview/observability-section.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: CodeStream observability
+metaDescription: "The observability section is where you connect CodeStream to your New Relic account so that you can take advantage of these observability features."
+---
+
+In order to get the most out of CodeStream and New Relic, connect CodeStream to New Relic via your [New Relic user key](/docs/apis/intro-apis/new-relic-api-keys/#user-api-key).
+
+Once that's done, for entities you're monitoring with New Relic, you'll see your errors directly in CodeStream. 
+
+## Connect CodeStream and New Relic [#connect]
+
+![A screenshot of the unconnected observability section where you add your New Relic user key.](./images/ObservabilitySection-connect.png "The unconnected observability section where you add your New Relic user key.")
+
+<figcaption>Before you can start seeing errors in your IDE and take advantage of other New Relic and CodeStream features, you'll need to enter your New Relic user key.</figcaption>
+
+Go here to get or create your [New Relic user key](https://one.newrelic.com/launcher/api-keys-ui.api-keys-launcher).
+
+Once you have your user key, in **Observability** click **Connect to New Relic One**, then paste your user key and click **Connect**.
+
+## See your errors in CodeStream [#errors]
```

Comment:
Feels like we need some more info here. The "Errors assigned to me" section is probably self evident, but we should note that the other sections represent recent errors for repos that you have open in your IDE. 

It was probably hard to see in the screenshot I gave you, but if one of the repos you have open in your IDE has remote URLs that map to multiple entities on NR1 there's a dropdown to the right of the "Recent errors in $reponame" heading where you can select the entity to filter by (e.g., you may want to see errors in production).

Lastly, maybe also worth noting that if the remotes for a given repo aren't associated with any entities on NR1 we give you the opportunity to make an association. That's depicted for the hello-ny repo in the screenshot.

Reply:
We should also let the user know what happens when you actually click on an error. That ties into a comment I left farther down. https://github.com/newrelic/docs-website/pull/4230/files#r732760743

Reply:
Yeah, it's tricky. I don't have an easy way to see this UI with live data. I'll add this content. Thanks!

Reply:
I've added some content to address your points here.